modules/assim.sequential/R/sda.enkf.R

```diff
@@ -133,29 +145,58 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
   X <- IC
   
   ## Load Parameters
-  if (sample_parameters == TRUE) {
-    settings$ensemble$size <- settings$state.data.assimilation$n.ensemble
+  if(is.null(restart) & is.null(restart$params)){
+    if (sample_parameters == TRUE) {
+      settings$ensemble$size <- settings$state.data.assimilation$n.ensemble
+    } else {
+      settings$ensemble$size <- 1
+    }
+    cumulative_ensemble_samples <- numeric(0)
+    repeat{ # temporary SIPNET hack, I want to make sure sum <1 for SIPNET
```

Comment:
This should at a minimum be generalized to something like ```if(model == 'SIPNET'){  repeat...```

Reply:
Are these checks that should be added to get.parameter.samples instead?

Reply:
this was a nasty hack, I probably should just comment it out, it also assumes the pft is `temperate.deciduous_SDA` 

I think in the end I moved towards defining the priors such that their sum won't exceed 1, but for now we can leave it as a comment as a reminder